pipeline/plot/unit_characteristic_plot.py

```diff
@@ -129,20 +129,33 @@ def plot_unit_selectivity(probe_insert_key, axs=None):
 
 
 def plot_unit_bilateral_photostim_effect(probe_insert_key, axs=None):
```

Comment:
the convention is to name the axes argument ax, and not axs. Minor point but would improve coherence with other libraries. In particular, axs should be reserved for the case where an array of Axes object is expected.

Reply:
even in the case of ax containing multiple axes

Reply:
Oops, did I misplace my comment? I meant this only for a plot that was producing single panel plot, not those that have multiple axes.